.pipelines/OneBranch.Official.yml

```diff
@@ -54,8 +54,7 @@ extends:
     - stage: linux_stage
       jobs:
       - job: linux_job
-        pool:
-          type: linux
+        pool: {vmImage: 'ubuntu-latest', name: 'Linux-CPU'}
```

Comment:
No... you need to use their machine pool.

Reply:
Ah... probably it is because the job itself is already in docker.

Reply:
can we ask onebranch team for a solution, this is a very common problem.

Reply:
I will give you their support mail list.